app/controllers/devise/two_factor_authentication_controller.rb

```diff
@@ -52,7 +52,7 @@ def handle_valid_otp
     warden.session(resource_name)[TwoFactorAuthentication::NEED_AUTHENTICATION] = false
 
     sign_in resource_name, resource, bypass: true
-    set_flash_message :notice, :success
+    flash[:notice] = t('devise.two_factor_authentication.success')
```

Comment:
I'm not sure what this was doing before... perhaps it was trying to clear the flash messages?  How was it not failing give that the translation didn't exist?